.github/workflows/manual-upgrade-tests.yaml

```
@@ -0,0 +1,18 @@
+        description: "Version to upgrade From"
+        required: false
+      target_version:
+        description: "Version to upgrade to"
+        required: false
+
+jobs:
+  nightly-upgrade-tests:
+    uses: ./.github/workflows/upgrade-tests-workflow.yaml
+    secrets: inherit
+    with:
+      branch: ${{ github.event.inputs.source_version }}
```

actionlint (1.7.7)

input "branch" is not defined in "./.github/workflows/upgrade-tests-workflow.yaml" reusable workflow. defined inputs are "source_version", "target_version"

(workflow-call)